src/Core/AbstractTest.php

```diff
@@ -182,6 +182,30 @@ public function setPending($state)
     }
 
     /**
+     * Set the focused status of the test and its children according to the
+     * supplied focus pattern and/or skip pattern
+     *
+     * @param string|null $focusPattern
+     * @param string|null $skipPattern
+     */
+    public function applyFocusPatterns($focusPattern, $skipPattern = null)
+    {
+        $title = $this->getTitle();
+
+        if ($skipPattern !== null && preg_match($skipPattern, $title)) {
```

Comment:
I think this looks good man. `\b` seems like a sane default if an explicit pattern is not given. Anything else in the pipeline for this? If not, I'll pull it down and give the CLI a go. If that’s all good I am good to merge this.